### PR TITLE
Mte simulation 20180510a

### PR DESCRIPTION
**Pull request by easye**
Aligns documentation with running the simulator.

Subsumes <https://github.com/emotiq/emotiq/pull/139> and <https://github.com/emotiq/emotiq/pull/138>.

### CODE REVIEW

File: src/Cosi-BLS/cosi-handlers.lisp [hunk 1]
```diff
@@ -1078,7 +1083,11 @@ bother factoring it with NODE-COSI-SIGNING."
 
 ;; ------------------------------------------------------------------------------------------------
 
-(defun leader-exec (node)
+(defun leader-exec (node &key (cosi-sign-prepare-timeout 10))
```
Comment (guitarvydas) [inline src/Cosi-BLS/cosi-handlers.lisp:1086]: Should probably be cosi-timeout (see below, I hope).  There are two hard-coded 10's in the original code.

Ah, I can't comment on code that you haven't changed... There is another hard-coded 10 in the code (ac:self-call :cosi-sign-commit self new-block 10) .  The 10 should be changed to cosi-timeout (I think - unless you did this on purpose).  Leader-exec sends itself "wait-prep-signing" which then gives itself a continuation and self-calls :cosi-sign-commit (with the 2nd hard-coded 10)

File: src/Cosi-BLS/cosi-handlers.lisp [hunk 2]
```diff
@@ -36,6 +36,10 @@ THE SOFTWARE.
 
 (defvar *current-node*  nil)  ;; for sim = current node running
 
+(defvar *cosi-sign-prepare-timeout* 10
```
Comment (guitarvydas) [inline src/Cosi-BLS/cosi-handlers.lisp:39]: *cosi-timeout* (see below)

File: src/node-sim.lisp [hunk 3]
```diff
@@ -2,15 +2,37 @@
  
 (in-package :emotiq/sim)
 
-(defun initialize (&key (leader-node "127.0.0.1"))
+(defun initialize (&key
+                     (executive-threads nil)
+                     (nodes 8)
+                     (new-configuration-p nil)
+                     (cosi-sign-prepare-timeout 10)
```
Comment (guitarvydas) [inline src/node-sim.lisp:9]: cosi-timeout

File: src/Cosi-BLS/cosi-handlers.lisp (hunk 1 above)
Comment (easye) [reply src/Cosi-BLS/cosi-handlers.lisp:1086]: @guitarvydas I changed the code to mimic what you had done in #139.   You only changed the first timeout.

**Pull request closed without merging**